packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -502,6 +509,11 @@ class TwinDeploymentHandler {
   }
 
   async handle(twinDeployments: TwinDeployment[]) {
+    const kycStatus = await this.kyc.status();
+    if (kycStatus !== KycStatus.verified)
+      throw new ValidationError(
+        "Your account is not verified. Please sign into Threefold Dashboard or Connect mobile app to complete your KYC verification.",
```

Comment:
Revise the app name it's called `Threefold Connect`

Reply:
replaced with ` Your account is not verified. Please sign into ThreeFold Dashboard or ThreeFold Connect mobile app to complete your KYC verification.`